regression/scripts/airfoil_import/airfoil_import_test.py

```diff
@@ -16,9 +18,25 @@ def main():
     airfoil_polar_names    = ['airfoil_polar_1.txt','airfoil_polar_2.txt']    
     airfoil_polar_data =  import_airfoil_polars(airfoil_polar_names) 
 
```

Comment:
Great Job! Make sure to add yourself as a contributor to the header of this script.